packages/extension/src/newtab/MainFeedPage.tsx

```diff
@@ -108,13 +135,42 @@ export default function MainFeedPage({
       screenCentered={false}
       customBanner={isDndActive && <DndBanner />}
       additionalButtons={
-        user && (
-          <SimpleTooltip content="Do Not Disturb" placement="bottom">
-            <HeaderButton
-              icon={<TimerIcon />}
-              className="btn-tertiary"
-              onClick={() => setShowDnd(true)}
-              pressed={showDnd}
+        placement === 'header' &&
+        !contentScriptGranted && (
+          <SimpleTooltip
```

Comment:
I feel like this can be it's own component as it will be re-used in the sidebar later on.

Reply:
Totally makes sense. Let me refactor this one! 🚢